ajax/libs/grommet/package.json

```diff
@@ -0,0 +1,107 @@
+{
+  "name": "grommet",
+  "filename": "grommet.min.js",
+  "npmName": "grommet",
+  "npmFileMap": [
+    {
+      "basePath": "bin",
```

Comment:
What's the reason why to set `"bin"` as basePath? If you use it as basePath, how did you get the css files?